lib/ansible/cli/__init__.py

```diff
@@ -422,6 +422,13 @@ def post_process_args(self, options):
                     skip_tags.add(tag.strip())
             options.skip_tags = list(skip_tags)
 
+        # Make sure path argument doesn't have a backslash
+        if hasattr(options, 'action') and (options.action == 'install' or options.action == 'download'):
+            if hasattr(options, 'args') and len(options.args) > 0:
```

Comment:
You should be able to avoid all the subsequent checks by using `var.rstrip("/")`

Reply:
Thanks - good point.